src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/PageLayout/DefaultPageLayout.constants.js

```diff
@@ -22,10 +22,6 @@ export const WidgetsForEnrollmentEventEdit: $ReadOnly<{ [key: string]: WidgetCon
 
 export const DefaultPageLayout: PageLayoutConfig = {
     leftColumn: [
-        {
-            type: WidgetTypes.COMPONENT,
-            name: 'TwoEventWorkspace',
-        },
```

Comment:
I think we need to remove "TwoEventWorkspace" from the available components as well: `src/core_modules/capture-core/components/Pages/common/EnrollmentOverviewDomain/EnrollmentPageLayout/DefaultEnrollmentLayout.types.js` as well.